### ISSUE TITLE
Accessibility Issue with launch link section in view.php page.

### ISSUE THREAD

**Issue by caperneoignis** [davidpesce/moodle-mod_tincanlaunch#148]
When on the view page of the mod plugin page. mod/tincanlaunch/view.php?id=xx the screen is not keyboard accessible. This means a person lacking the ability to use a mouse or touch screen is unable to access any of the links, using the tab key and enter key. This is a 508 issue that will need to be rectified to conform to 508 requirements. 

I maybe able to do a pull request for this to help fix the issue, unless someone is working on it. 

The affected version is 1.4 with Moodle 3.4.1+.

Thanks for your time.

**Comment by garemoko** [comment]
Thanks for testing!

I'm not aware of anybody already working on this. 

Changing the name of the issue from "508" to "accessibility" as 508 is a US specific law and this issue isn't specific to accessibility in the US.

**Comment by caperneoignis** [comment]
Oh yea, that's a good catch. Forgot 508 wasn't an international standard.

**Comment by caperneoignis** [comment]
I will start work on this the week after next, since my project is using this plugin, I have this listed as an issue in our sprint board. I will ensure to open a pull request once we have tested the plugin though accessibility test. This will be the 508 standard, so if someone else, sees WCAG issues I'll try and include those fixes if they are not too difficult. But I haven't seen any so far, but we mainly do 508 because of contract requirements with some WCAG checks thrown in.

**Comment by garemoko** [comment]
I'm closing this because I don't plan to work on it and I'm tidying the issues list. PR still very welcome though.
